gulpfile.js

```diff
@@ -65,6 +67,9 @@ gulp.task('pug', () => {
 gulp.task('styl', () => {
 	return gulp.src('src/styl/index.styl')
 		.pipe(stylus({compress: true}))
+		.pipe(postcss([
+			autoprefixer({browsers: ['last 1 version']})
+    	]))
```

Comment:
Looks like there might be some spaces here... or github is messing with me

Reply:
It is fine in editor, maybe github compare make it look weird.